src/com/comeet/Room.java

```diff
@@ -161,7 +161,7 @@ public void addFreeBusyTime(FreeBusySlot slot) {
     }
 
     @XmlElement(name = FREEBUSY)
-    public List<FreeBusySlot> getFreeBusy() {

```

Comment:
That's what I thought too but I went through it with Ricardo it still had the incorrect property values